src/multicall.ts

```diff
@@ -596,6 +596,7 @@ export class Multicall {
       case Networks.kcc:
       case Networks.lineaTestnet:
       case Networks.linea:
+      case Networks.scroll:
```

Comment:
Scroll Alpha multicall deployment is at `0x2117f703867a2B7E6813c7e5Edd96bf9a8d8eC30`

Reply:
And also should be renamed to Networks.scrollAlpha